src/nunavut/lang/__init__.py

```diff
@@ -231,13 +231,23 @@ def support_files(self) -> typing.Generator[pathlib.Path, None, None]:
         """
         Iterates over non-templated supporting files embedded within the Nunavut distribution.
         """
-        module = importlib.import_module('nunavut.lang.{}.support'.format(self._language_name))
+        try:
+            module = importlib.import_module('nunavut.lang.{}.support'.format(self._language_name))
+
+            # All language support modules must provide a list_support_files method
+            # to allow the copy generator access to the packaged support files.
+            list_support_files = getattr(module, 'list_support_files')  \
+                # type: typing.Callable[[], typing.Generator[pathlib.Path, None, None]]
+            return list_support_files()
+        except ModuleNotFoundError:
+            # No serialization support for this language
+            logger.warn("No serialization support for selected target. Skipping serialization support generation.")
```

Comment:
this is a much better strategy but I'd make this an info message. If the language doesn't have serialization support then that's the nature of that language. There's nothing to warn the user about in such a case.